packages/react/package.json

```diff
@@ -33,7 +33,7 @@
     "format:diff": "prettier --list-different \"**/*.{scss,css,js,jsx,md,ts}\"",
     "lint": "yarn lint:javascript && yarn lint:stylelint",
     "lint:javascript": "eslint --ext .jsx --ext .js .",
-    "lint:stylelint": "stylelint './src/**/*.scss' --syntax scss --ignorePath .gitignore --custom-formatter ./config/stylelint/sassMsgFormatter.js",
+    "lint:stylelint": "stylelint './src/**/*.scss' --ignorePath .gitignore --custom-formatter ./config/stylelint/sassMsgFormatter.js",
```

Comment:
syntax is removed in newer versions of stylelint. Upgrade necessary to support `@use`